Library/Homebrew/cmd/pull.rb

```diff
@@ -516,7 +516,11 @@ def verify_bintray_published(formulae_names)
           opoo "Cannot publish bottle: Failed reading info for formula #{f.full_name}"
           next
         end
-        bottle_info = jinfo.bottle_info(jinfo.bottle_tags.first)
+        if f.tap.remote.include?("Linuxbrew") && jinfo.bottle_tags.include?("x86_64_linux")
```

Comment:
Agreed that it'd be nice. Will spin off its own issue for it though, since otherwise I'd be changing a lot more stuff on this PR than I'd like to.